test/charts/sparkline/sparkline-chart.spec.js

```diff
@@ -105,7 +105,7 @@ describe('Directive: pfSparklineChart', function() {
 
     expect(isolateScope.config.data.x).toBe("dates");
     expect(isolateScope.config.data.columns.length).toBe(2);
-    expect(isolateScope.config.data.columns[0][1]).toBe($scope.data.xData[1]);
+    expect(isolateScope.config.data.columns[0][1].toString()).toBe($scope.data.xData[1].toString());
```

Comment:
I believe the unit test was checking that the actual javascript objects were the same, after the change:
+     scope.config = angular.merge({}, scope.defaultConfig, scope.config);
-      scope.config = $.extend(true, angular.copy(scope.defaultConfig), scope.config);
They don't appear to be, even though their contents are the same.  To me the angular.merge and the $.extend cmds should result in new javascript objects, so I'm not sure why/what is different between the commands.  In other words, I wouldn't think the 'toString()' would be neccesary.